financial-connections-lite/res/values/strings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="stripe_fc_lite_exit_title">Are you sure you want to exit?</string>
+    <string name="stripe_fc_lite_exit_message">You haven\'t finished linking you bank account and all progress will be lost.</string>
```

Comment:
```suggestions
    <string name="stripe_fc_lite_exit_message">You haven\'t finished linking your bank account and all progress will be lost.</string>
```

Reply:
good catch, the same exists on iOS @mats-stripe - https://github.com/stripe/stripe-ios/blob/62dc2874f1a9/StripePaymentSheet/StripePaymentSheet/Source/Internal/FC%20Lite/Controllers/FCLiteContainerViewController.swift#L260